test/rgb.js

```diff
@@ -754,10 +827,11 @@ exports["RGB"] = {
   },
 
   "Animation.render": function(test) {
-    test.expect(1);
-    this.color = this.sandbox.stub(this.rgb, "color");
-    this.rgb[Animation.render]([0]);
-    test.equal(this.color.callCount, 1);
+    test.expect(2);
+    // rgb.write() is already wrapped
```

Comment:
where is rgb.write() wrapped?

Reply:
Line 55, in the `setUp` method for this block of tests, it's stubbed using sinon:

https://github.com/code-dot-org/johnny-five/blob/f1b1a55634ab068ca9abef8c60f7236a26045643/test/rgb.js#L55